app/routes/show-project.js

```diff
@@ -6,6 +6,47 @@ export default class ShowProjectRoute extends Route {
     return this.store.findRecord('project', id, { reload: true });
   }
 
+  afterModel(...args) {
+    const [model] = args;
+
+    model.actions.forEach((cpcAction) => {
+      if (cpcAction.statuscode === 'Approved') {
+        if (cpcAction.actioncode === 'BD'
```

Comment:
Logic is: if it's approved _and_ the actioncode is one of a list of these? 

Here's one way to do it: 

```
const actionCodes = ['BD', 'HI', 'HK', /* ... more */];
if (cpcAction.statuscode === 'Approved' &&  actionCodes.includes(cpcAction.actioncode)) {
  // other
}
```